src/WebJobs.Script/runtimeassemblies-v3.json

```diff
@@ -0,0 +1,1439 @@
+    {
+      "name": "System.Text.Encodings.Web",
+      "resolutionPolicy": "minorMatchOrLower"
+    },
+    {
+      "name": "System.Text.RegularExpressions",
+      "resolutionPolicy": "minorMatchOrLower"
+    },
+    {
+      "name": "System.Threading",
+      "resolutionPolicy": "minorMatchOrLower"
+    },
+    {
+      "name": "System.Threading.Channels",
+      "resolutionPolicy": "minorMatchOrLower"
+    },
+    {
+      "name": "System.Threading.Overlapped",
+      "resolutionPolicy": "minorMatchOrLower"
+    },
+    {
+      "name": "System.Threading.Tasks",
+      "resolutionPolicy": "minorMatchOrLower"
+    },
+    {
+      "name": "System.Threading.Tasks.Dataflow",
+      "resolutionPolicy": "minorMatchOrLower"
+    },
+    {
+      "name": "System.Threading.Tasks.Extensions",
+      "resolutionPolicy": "minorMatchOrLower"
+    },
+    {
+      "name": "System.Threading.Tasks.Parallel",
+      "resolutionPolicy": "minorMatchOrLower"
+    },
+    {
+      "name": "System.Threading.Thread",
+      "resolutionPolicy": "minorMatchOrLower"
+    },
+    {
+      "name": "System.Threading.ThreadPool",
+      "resolutionPolicy": "minorMatchOrLower"
+    },
+    {
+      "name": "System.Threading.Timer",
+      "resolutionPolicy": "minorMatchOrLower"
+    },
+    {
+      "name": "System.Transactions",
+      "resolutionPolicy": "minorMatchOrLower"
+    },
+    {
+      "name": "System.Transactions.Local",
+      "resolutionPolicy": "minorMatchOrLower"
+    },
+    {
+      "name": "System.ValueTuple",
+      "resolutionPolicy": "minorMatchOrLower"
+    },
+    {
+      "name": "System.Web",
+      "resolutionPolicy": "minorMatchOrLower"
+    },
+    {
+      "name": "System.Web.HttpUtility",
+      "resolutionPolicy": "minorMatchOrLower"
+    },
+    {
+      "name": "System.Windows",
+      "resolutionPolicy": "minorMatchOrLower"
+    },
+    {
+      "name": "System.Xml",
+      "resolutionPolicy": "minorMatchOrLower"
+    },
+    {
+      "name": "System.Xml.Linq",
+      "resolutionPolicy": "minorMatchOrLower"
+    },
+    {
+      "name": "System.Xml.ReaderWriter",
+      "resolutionPolicy": "minorMatchOrLower"
+    },
+    {
+      "name": "System.Xml.Serialization",
+      "resolutionPolicy": "minorMatchOrLower"
+    },
+    {
+      "name": "System.Xml.XDocument",
+      "resolutionPolicy": "minorMatchOrLower"
+    },
+    {
+      "name": "System.Xml.XmlDocument",
+      "resolutionPolicy": "minorMatchOrLower"
+    },
+    {
+      "name": "System.Xml.XmlSerializer",
+      "resolutionPolicy": "minorMatchOrLower"
+    },
+    {
+      "name": "System.Xml.XPath",
+      "resolutionPolicy": "minorMatchOrLower"
+    },
+    {
+      "name": "System.Xml.XPath.XDocument",
+      "resolutionPolicy": "minorMatchOrLower"
+    },
+    {
+      "name": "System.Xml.XPath.XmlDocument",
+      "resolutionPolicy": "minorMatchOrLower"
+    },
+    {
+      "name": "WindowsBase",
+      "resolutionPolicy": "minorMatchOrLower"
+    }
+  ]
+}
```

Comment:
It looks like Newtonsoft.Json is removed from this already. Could you do one commit where you copied v2 exactly, then follow up with one that changes things for v3? It'd give us better history of what'd actually changed...

Reply:
Sure.... I can do that